config/config.yaml

```diff
@@ -18,6 +18,8 @@ kraken:
         -   litmus_scenarios:                              # List of litmus scenarios to load
             - - https://hub.litmuschaos.io/api/chaos/1.10.0?file=charts/generic/node-cpu-hog/rbac.yaml
               - scenarios/node_hog_engine.yaml
+        -   cluster_shut_down_scenario:
```

Comment:
The scenario type in run_kraken has an "s" at the end of the scenario. Just need to add an s here to be: **cluster_shut_down_scenarios**